pandas/io/excel/_odfreader.py

```diff
@@ -60,7 +61,7 @@ def get_sheet_by_name(self, name: str):
             if table.getAttribute("name") == name:
                 return table
 
-        raise ValueError("sheet {name} not found".format(name))
+        raise ValueError("sheet {name} not found".format(name=name))
```

Comment:
closes #27676

Reply:
This is being address in #27677 with more detail (i.e. tests) so would rather revert here and let that PR take care of it, especially since it is their first contribution